drivers/include/pulse_counter.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_pulse_counter sensor
+ * @ingroup     drivers_sensors
+ *
+ * The connection between the MCU and the PULSE_COUNTER is based on the
+ * GPIO-interface.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Driver for the PULSE_COUNTER sensor
+ *
+ * @author      Hyung-Sin <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef PULSE_COUNTER_H_
```

Comment:
Please remove the trailing underscore, that's why the CI on Travis is broken (same at the end of this file)

Reply:
Thanks, I addressed it! :)